payjoin-cli/src/app.rs

```diff
@@ -194,6 +194,25 @@ impl App {
         Ok(())
     }
 
+    #[cfg(feature = "v2")]
+    async fn attemp_v1(&self, req_ctx: &mut payjoin::send::RequestContext) -> Result<Psbt> {
+        let (req, ctx) = req_ctx.clone().extract_v1().unwrap();
+        let http = http_agent()?;
+        println!("Sending fallback v1 request to {} after v2 request attempt fail", &req.url);
+        let response = spawn_blocking(move || {
+            http.post(req.url.as_ref())
+                .set("Content-Type", "text/plain")
+                .send_bytes(&req.body)
+                .map_err(map_ureq_err)
+        })
+        .await??;
+        println!("Sent fallback v1 request");
+        ctx.process_response(&mut response.into_reader()).map_err(|e| {
+            log::debug!("Error processing response: {:?}", e);
+            anyhow!("Failed to process response {}", e)
```

Comment:
we dont really handle the error here, but that might be too much error handling? what do you think?

Reply:
Right now on line 237 the error for v2 still gets printed if the retry happens. It seems logical to me to handle the error by retrying INSTEAD of returning, and then handle both v1 and v2 ctx.process_response errors with the same piece of code after if you still end up with an error. I *think* I'd just leave `ctx.process_response(&mut response.into_reader())` to be returned here instead of printing. Do all the printing and merged error handling in `long_poll_post`